transform/transform.go

```diff
@@ -27,16 +29,18 @@ type transformOperationJSON struct {
 	Args map[string]string `json:"args"`
 }
 
-// transformInstruction defines a JSONPath for a transform and an optional set of operations to be performed on the
+// transformInstruction defines a JSONPath and XMLPath for a transform and an optional set of operations to be performed on the
 // data from that path.
 type transformInstruction struct {
 	// For JSONPath format see http://goessner.net/articles/JsonPath/
-	jsonPath   string
+	JSONPath   string
```

Comment:
Looks like these could both be combined into just `path` since you're passing around the format to the transformer

Reply:
also I think I agree with Anderson's comment

Reply:
also - we went from private to public. Which is probably also wrong.

Reply:
also the comment for XPath is next to the JSONpath one :P